recipes/enlopy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "enlopy" %}
+{% set version = "0.1.dev11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f18898f20017f1159a74c21f593e7b3df5656da83836baa841e27b0136acdea
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=3.3
+  run:
+    - matplotlib >=2.0
+    - numpy >=1.12
+    - pandas >=0.20
+    - python
+    - scipy >=1.0
+
+test:
+  imports:
+    - enlopy
+
+about:
+  home: https://github.com/kavvkon/enlopy
+  license: BSD-3-Clause
+  license_family: BSD
+  summary: Python library with methods to generate, process, analyze, and plot energy related timeseries.
+  doc_url: https://enlopy.readthedocs.io/
+  dev_url: https://www.github.com/kavvkon/enlopy
```

Comment:
Please add the ``license_file``
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=license_file#about

Reply:
ok it is packaged in the tarball